SECURITY.md

```diff
@@ -42,6 +42,14 @@ RPC DoS/Crashes:
 $5,000 USD in locked SOL tokens (locked for 12 months)
 * RPC attacks
 
+Out of Scope:
+The following components are out of scope for the bounty program
+* Metrics: `/metrics` in the monorepo as well as https://metrics.solana.com
+* Explorer: `/explorer` in the monorepo as well as https://explorer.solana.com
+* Any credentials, auth tokens, etc. checked into the repo
```

Comment:
Maybe this?   If there's actual valid/accessible creds somehow I'd want to hear about them
```suggestion
* Any encrypted credentials, auth tokens, etc. checked into the repo
```